docs/getting-started/plugins.md

```diff
@@ -84,7 +128,7 @@ If a plugin execution is a prerequisite for another plugin, it must be located e
 
 Amplication plugin developers may release several versions of a plugin over time. By default, the latest version will be available when installing the plugin.
 
-Sometimes, you might need to generate a project with an earlier version of the plugin, to ensure compatibility.
+Sometimes, you might need to generate a project with an earlier version of the plugin, to ensure version compatibility.
```

Comment:
@dericksozo @mulygottlieb I think here it is wrong to say "generate a project"...maybe it should be "generate code" or "build code". thoughts?